code/modules/admin/admin_verbs.dm

```diff
@@ -188,7 +189,7 @@ GLOBAL_PROTECT(admin_verbs_debug)
 	/client/proc/cmd_display_init_log,
 	/client/proc/cmd_display_overlay_log,
 	/client/proc/reload_configuration,
-	/client/proc/give_all_spells,
+	///client/proc/give_all_spells,
```

Comment:
Does this no longer work or is it removed?

Reply:
Moved to a new panel idk how to make the panel appear by a button press

Reply:
Sadly I can't seem to give the command to give ALL spells because that freezes my pc :(